build/Dockerfile.ci

```diff
@@ -110,16 +123,19 @@ ENV OPERATOR=/usr/local/bin/windows-machine-config-operator \
 # Changes needed for our CI
 
 # install make package
-RUN microdnf -y install make tar.x86_64 gzip.x86_64 wget.x86_64
+RUN microdnf -y install make tar.x86_64 gzip.x86_64
 RUN microdnf -y update
 
-# Download and install Go. We need
-RUN curl -L -s https://dl.google.com/go/go1.13.8.linux-amd64.tar.gz > go1.13.8.linux-amd64.tar.gz \
-    && sha256sum go1.13.8.linux-amd64.tar.gz \
-    && echo "0567734d558aef19112f2b2873caa0c600f1b4a5827930eb5a7f35235219e9d8  go1.13.8.linux-amd64.tar.gz" | sha256sum -c \
-    && tar -xzf go1.13.8.linux-amd64.tar.gz \
-    && mv go /usr/local \
-    && rm -rf ./go*
+# Copy go from build stage, install within this stage
+COPY --from=build /build/go/go.tar.gz .
+RUN tar -C /usr/local -xzf go.tar.gz && rm go.tar.gz
+ENV PATH=${PATH}:/usr/local/go/bin
+# Allow building binaries and creation of /go/.cache directory by the hack script
+RUN go version
+RUN mkdir /go
+RUN chmod -R g=u+w /go
+ENV GOPATH="/go"
+ENV GOCACHE="/go/.cache"
 
 # Download and install oc
```

Comment:
We should move this to the download stage at some point.

Reply:
I agree, I was doing that, but its proved to be difficult due to CI image building issues. I reverted that change.